tests/conftest.py

```
@@ -488,6 +488,18 @@ def __init__(
 nio_mock.SyncError = MockSyncError
 
 
+class MockRoomSendError(Exception):
+    def __init__(
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)